go-controller/pkg/ovn/policy.go

```diff
@@ -125,7 +129,7 @@ func (oc *Controller) updateStaleDefaultDenyACLNames(npType knet.PolicyType, gre
 	}
 	// loop through the cleanUp map and per namespace update the first ACL's name and delete the rest
 	for namespace, aclList := range cleanUpDefaultDeny {
-		newName := namespacePortGroupACLName(namespace, "", gressSuffix)
+		newName := trimNameRFC1123(namespacePortGroupACLName(namespace, "", gressSuffix))
```

Comment:
@trozet : do you think I should trim this to 45 and then add the suffix? It seems like in `BuildACLs` we actually trim it in the very end so we loose out all the suffixes we add..

Reply:
nvm on second thoughts I realize having this as is ; is better since the 63 chars will need to be unique as it comes from namespaces, that's a better way to go than trimming it and causing conflicts.